contrib/syncer/syncer.yaml

```diff
@@ -0,0 +1,125 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kcp-system 
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: syncer
+  namespace: kcp-system 
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: syncer
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - create
+- apiGroups:
+  - ""
+  # TODO set only the resources needed for the syncer
+  resources:
+  - services
+  - deployments
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - delete
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: syncer
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: syncer
+subjects:
+- kind: ServiceAccount
+  name: syncer
+  namespace: kcp-system
+---
+# Secret with the kubeconfig used to connect to the Kcp server
+apiVersion: v1
+kind: Secret
+metadata:
+  name: syncer
+  namespace: kcp-system
+data:
+  kubeconfig: |-
+    apiVersion: v1
+    kind: Config
+    clusters:
+    - name: default-cluster
+      cluster:
+        certificate-authority-data: KCP_CERTIFICATE_AUTHORITY
+        server: KCP_SERVER_URL
+    contexts:
+    - name: default-context
+      context:
+        cluster: default-cluster
+        namespace: default
+        user: default-user
+    current-context: default-context
+    users:
+    - name: default-user
+      user:
+        token: KCP_TOKEN
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: kcp-syncer-ID
+  name: syncer
+  namespace: kcp-system
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: kcp-syncer-ID
+  template:
+    metadata:
+      labels:
+        app: kcp-syncer-ID
+    spec:
+      containers:
+      - args:
+        - --from-kubeconfig=/kcp/kubeconfig
+        - --workload-cluster-name=WORKLOAD_CLUSTER_NAME   # ID of the -to cluster. Resources with this ID set in the 'workloads.kcp.dev/cluster' label will be synced
+        - --from-cluster=LOGICAL_CLUSTER                  # Name of the -from logical cluster
```

Comment:
Maybe note that this value can be found in the ClusterName field of the target workspace?